src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -48,7 +48,7 @@ object TSVFormatter {
   private def makeRow(entity: Entity, headerValues: IndexedSeq[String]): IndexedSeq[String] = {
     val rowMap: Map[Int, String] =  entity.attributes map {
       case (attributeName, attribute) =>
-        val columnPosition = headerValues.indexOf(attributeName.name)
+        val columnPosition = headerValues.indexOf(AttributeName.toDelimitedName(attributeName))
```

Comment:
this is the functional change in this PR - it will now look for "namespace"foo instead of just "foo" in the list of headers. Everything else in this PR is unit tests.